### ISSUE TITLE
getting RawData missing Error

### ISSUE THREAD

**Issue by saikumaradepu11** [MITESHPUTHRANNEU/Speech-Emotion-Analyzer#46]
mylist= os.listdir('RawData/')

getting the error as FileNotFoundError.
please let me know if anyone know how to solve this error and also guide me if where i need to place the dataset

**Comment by ruXlab** [comment]
It's a duplicate of #45

**Comment by vanimirg** [comment]
I'm getting the same error

**Comment by Chen840807** [comment]
sorry i have same question  can't find RawData this file...

**Comment by shanjng** [comment]
Did anyone figure it out?

**Comment by eknoor137** [comment]
Is this issue solved yet? any solution?

**Comment by shanjng** [comment]
@eknoor137 Off the top of my head, I believe I just ended up using my own wav file, disregarding the output10.wav, or there is another example wav file not named output10.wav but it's provided in the project

**Comment by MITESHPUTHRANNEU** [comment]
@all Sorry, I have been busy lately. This error relates to audio file location. 

This repository doesn't include the audio files used for training purpose. The links to download them has been provided in the readme file. Just create a new folder (any name). In this case I had created 'RawData' and dumped all the audio files into this folder. 

'output10.wav' was a sample audio I used to test my trained model and check it's prediction.
